docs/docs/how-tos/manual-backup.md

```diff
@@ -175,6 +175,27 @@ gsutil cp 2021-04-23.tar gs://<your_bucket_name>/backups/2021-04-23.tar
 cd /data
 gsutil cp gs://<your_bucket_name>/backups/2021-04-23.tar .
 ```
+### Azure
+# install Azure CLI
+curl -sL https://aka.ms/InstallAzureCLIDeb |  bash
+az login --use-device-code
```

Comment:
I think the CI failures might resolve if we add code block quotes around these and the codes below